clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -7209,6 +7210,32 @@ void OffloadWrapper::ConstructJob(Compilation &C, const JobAction &JA,
       if (A->getValue() == StringRef("image"))
         WrapperArgs.push_back(C.getArgs().MakeArgString("--emit-reg-funcs=0"));
     }
+    // Grab any Target specific options that need to be added to the wrapper
+    // information.
+    ArgStringList BuildArgs;
+    auto createArgString = [&](const char * Opt) {
+      if (!BuildArgs.empty()) {
```

Comment:
Does an early empty `return` affect the `void` return type's deducing? In the event it works, an early exit would be preferable.

Reply:
Early exit looks to work okay and made changes to reflect.